README.md

```diff
@@ -10,46 +10,70 @@ _Note_ - the readme is currently being written and is very incomplete
 
 ## Overview
 
-After installing this extension double click on a FITS file in jupyterlab and see the image in a tab.
+This extension adds the following features to JupyterLab:
+
+  - Double click on a FITS file and see it in a tab
+  - Start the full Firefly viewer in a tab.
+  - use `FireflyClient` in a python notebook to start Firefly in a tab and send data
+  (tables,images,charts) to it using the `FireflyClient` API
+  - The are several Widgets (images,tables,charts) that can be used in a notebook
+  - The `SlateWidget` is a full firefly viewer widget the provides a `FireflyClient` instance to embed a full firefly in a notebook as a widget
 
 
 
 ## Prerequisites
 
-* JupyterLab ^0.28.0
+* JupyterLab ^0.35.1
 
 
 
-### Before running
+### Before running setup the firefly URL - 3 ways
 
-Add the following line to your `~/.jupyter/jupyter_notebook_config.py`
+ * Add the following line to your `~/.jupyter/jupyter_notebook_config.py`
 
-```python
-c.Firefly.url = 'http://localhost:8080/firefly'
-```
+   ```python
+   c.Firefly.url = 'http://localhost:8080/firefly'
```

Comment:
I think this might have something to do with my recent upgrade to Mac OS X Mojave: I had to use `127.0.0.1` instead of `localhost`.

Reply:
Might just be something to add to the README.